yaki_admin/src/features/captain/services/isActiveTeam.ts

```diff
@@ -0,0 +1,22 @@
+import {reactive} from "vue";
+
+/**
+ * Get the selected team from @click in v-for SideBarCaptainContent.
+ * In TeamListElement compare the selected Team with the element teamName
+ * If thoses are the same return true else false.
+ * Apply css class if true, if false its not applied.
+ */
+const isTeamSelected = reactive({
+  teamName: "Team 1",
+  setTeam(team: string) {
+    this.teamName = team;
+  },
+  isSameIndex(team: string) {
```

Comment:
method name incorrect should be changed later when proper team fetch implementation